apps/nextjs-website/react-components/components/MegaHeader/MegaHeader.tsx

```diff
@@ -59,8 +59,26 @@ const MegaHeader = (props: MegaHeaderProps) => {
     };
   }, []);
 
+  useEffect(() => {
+    if (mobileMenuOpen) {
+      document.body.style.overflow = 'hidden';
```

Comment:
```suggestion
      document.body.style.overflow = 'hidden';
      document.body.style.touchAction = 'none';
```
I believe Safari on iOS ignores overflow hidden applied to the body.
Removing touchAction should fix the issue.